packages/bezier-react/src/components/Modals/Modal/ModalContent.tsx

```diff
@@ -47,28 +44,29 @@ export const ModalContent = forwardRef(function ModalContent({
 
   return (
     <DialogPrimitive.Portal container={container}>
-      <Styled.DialogPrimitiveOverlay />
-      <DialogPrimitive.Content asChild>
-        <Styled.Content
-          aria-modal
-          ref={forwardedRef}
-          style={contentStyle}
-          {...rest}
-        >
-          <Styled.Section>
-            <ModalContentContext.Provider value={contextValue}>
-              { children }
-            </ModalContentContext.Provider>
+      <Styled.DialogPrimitiveOverlay style={{ '--bezier-modal-z-index': zIndex } as React.CSSProperties}>
```

Comment:
```suggestion
      const overlayStyle = useMemo(() => ({
        '--bezier-modal-z-index': zIndex,
      }, [zIndex])
      
      <Styled.DialogPrimitiveOverlay style={overlayStyle}>
```

큰 의미는 없지만, 얘도 겸사겸사 memo해주는 게 낫지 않을까 제안해봅니당